BUILD.gn

```diff
@@ -191,6 +191,10 @@ if (current_toolchain != "${dir_pw_toolchain}/default:default") {
 
     # Set this to true to enable k32w builds by default.
     enable_k32w_builds = false
+
+    # Various cross builds
+    enable_arm_clang_crossbuild = false
```

Comment:
enable_linux_arm_build

Cross compiling is just a technique, generally we should avoid caring about that.